### PR TITLE
Address some issues found by sonarcloud

### PR DESCRIPTION
**Pull request by TrentHouliston**

### CODE REVIEW

**Review by TrentHouliston — COMMENTED** [review]
Placeholder so I can see since last review

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@ add_library(NUClear::nuclear ALIAS nuclear)
 # Set compile options for NUClear
 target_link_libraries(nuclear ${CMAKE_THREAD_LIBS_INIT})
 set_target_properties(nuclear PROPERTIES POSITION_INDEPENDENT_CODE ON)
-target_compile_features(nuclear PUBLIC c_std_99 cxx_std_14)
+target_compile_features(nuclear PUBLIC cxx_std_14)
```
Comment (Bidski) [inline src/CMakeLists.txt:38]: SonarQube complains about this? OR is this part of the xxhash changes?

**Review by Bidski — CHANGES_REQUESTED** [review]

File: src/CMakeLists.txt (hunk 1 above)
Comment (TrentHouliston) [reply src/CMakeLists.txt:38]: It was a part of xxhash, the c files are gone

**Review by Bidski — APPROVED** [review]
LGTM

**Pull request merged**